src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,102 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_SIZE = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int MULTIPLIER_FOR_SIZE = 2;
+    private Node<K, V>[] table;
+    private int size;
+    private int threshold;
+
+    MyHashMap() {
+        table = (Node<K, V>[]) new Node[DEFAULT_SIZE];
+        threshold = (int) (table.length * LOAD_FACTOR);
```

Comment:
```suggestion
        threshold = (int) table.length * LOAD_FACTOR;
```

Reply:
https://pastenow.ru/164ca44bdb9c84db9d983e2fb01d523c
It doesn't work like that